server/src/get_server.ts

```diff
@@ -6,6 +6,10 @@ import { CONFIG } from './config'
 import { Router } from './router'
 import { TasksAndModels } from './tasks'
 
+// env variable used by discojs in order to be able to load
+// models through the file system.
+process.env.TFJSNODE = '@tensorflow/tfjs-node'
```

Comment:
why do we need to setup an env var here? seems like a lot of trouble for something that could be a simple var :p

Reply:
nice